### ISSUE TITLE
what's going on here

### ISSUE THREAD

**Issue by hvgazula** [hvgazula/trained-models#3]
@gaiborjosue any clue what is going on here? https://github.com/hvgazula/trained-models/actions/runs/6763667306/job/18381114824#step:9:740

**Comment by gaiborjosue** [comment]
It is an error related to pushing to OSF

**Comment by hvgazula** [comment]
hmm...but you never had that issue..did you?

**Comment by gaiborjosue** [comment]
No, because all the times I tested the workflow with files, I commented out the line that pushes it to osf, since I was just testing.

**Comment by hvgazula** [comment]
ah okay...well looks like i have to first test it then...however I did try it this morning and it all went well. see this https://github.com/neuronets/test-datalad-osf/actions/runs/6761803498

**Comment by hvgazula** [comment]
in fact, as you can see i uploaded a ckpts.zip file which is as good as sample.zip (🤷‍♂️ ) what do you think? well..let me see what I can do.

**Comment by gaiborjosue** [comment]
Mmm I see. Then it is probably an issue with EC2 itself. Probably just make a workflow really simple:

1. Start ec2
2. Upload to osf from ec2
3. Stop ec2.

Just to check it that will work.

**Comment by hvgazula** [comment]
aye aye

**Comment by hvgazula** [comment]
i think it is failing here

          ```
sudo apt install python3-pip -y
          sudo apt-get install datalad -y
          python3 -m pip install datalad-osf
          sudo python3 -m pip install datalad-installer
          datalad-installer --sudo ok git-annex -m datalad/git-annex:release
          sudo git config --global filter.annex.process "git-annex filter-process"

          git config --system user.name "trained_models"
          git config --system user.email "trained_models"
```

Remember this? Here's mine https://github.com/hvgazula/trained-models/actions/runs/6764535767/job/18383054481 and yours is here https://github.com/gaiborjosue/trained-models-fork/actions/runs/6714213901/workflow#L472
